osquery/utils/system/windows/env.cpp

```diff
@@ -54,17 +60,20 @@ boost::optional<std::string> getEnvVar(const std::string& name) {
   // returned size is greater than what we expect.
   if (value_len > kInitialBufferSize) {
     buf.assign(value_len, '\0');
-    value_len = ::GetEnvironmentVariableA(name.c_str(), buf.data(), value_len);
-    if (value_len == 0 || value_len > buf.size()) {
-      // The size returned is greater than the size we expected. Currently, we
-      // will not deal with this scenario and just return as if an error has
-      // occurred.
-      return boost::none;
-    }
+    value_len =
+        ::GetEnvironmentVariableW(widename.c_str(), buf.data(), value_len);
   }
 
-  return std::string(buf.data(), value_len);
-}
+  if (value_len == 0 || value_len > buf.size()) {
+    // The size returned is greater than the size we expected. Currently, we
+    // will not deal with this scenario and just return as if an error has
+    // occurred.
+    return boost::none;
+  }
+
+  return wstringToString(buf.data());
+
+} // namespace osquery
```

Comment:
nitpick, this comment looks like the result of a stray copy-paste

Reply:
Thanks! Updated.